maventa/maventa.ts

```diff
@@ -0,0 +1,95 @@
+import https from 'https';
+import { getAccessToken } from './auth';
+import config from './config';
+import Invoice from './types/invoice';
+
+// Send company invoice dependencies
+
+// import fs from 'fs';
+// import FormData from 'form-data';
+
+export async function listInvoices(): Promise<Invoice[]> {
```

Comment:
Actually create a (static) class like our other HTTP libraries. Not just functions.